### DISCUSSION TITLE
Welcome to Lavalite CMS Discussions!

### DISCUSSION THREAD

**Discussion by georgemjohn** [LavaLite/cms#357] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alagoroChapela** [comment]
I supose this is a good cms, but I only can supose, as installing instructios don´t work so start fixing installation, NOTHING WORKS if 10.x installation doesn´t work.

`composer create-project LavaLite/cms lavalitecms`
`php artisan lavalite:install
`
`SQLSTATE[42S02]: Base table or view not found: 1146 Table 'lavalitecms.menus' doesn't exist (Connection: mysql, SQL: alter table `menus` add `id` int unsigned not null auto_increment primary key, add `parent_id` int null default '0', add `key` varchar(100) null, add `url` varchar(100) null, add `icon` varchar(100) null, add `permission` varchar(1000) null, add `role` varchar(1000) null, add `name` text null, add `description` text null, add `target` enum('_blank', '_self') null, add `order` int null, add `uload_folder` varchar(150) null, add `slug` varchar(200) null, add `status` enum('Show', 'Hide') null default 'Show', add `upload_folder` varchar(100) null, add `deleted_at` timestamp null, add `created_at` timestamp null, add `updated_at` timestamp null)`

**Comment by Tomirad** [comment]
Still not working correct.
